packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.tsx

```diff
@@ -87,18 +88,17 @@ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, {
             height: "20px",
             marginRight: "10px",
         },
-        panelContent: {
-            [`& .${gridClasses.filterForm}:first-child .${gridClasses.filterFormLinkOperatorInput}`]: {
-                ["@media (max-width: 900px)"]: {
-                    display: "none",
-                },
-            },
-        },
```

Comment:
Is it really okay to remove this?

Reply:
I have changed it to only show the spacing on bigger breakpoints

Reply:
But the previous change hid the input altogether. I guess this was intentional on mobile? @jamesricky do you know something about this?

Reply:
Yes, this is the select to choose between `And`/`Or` which is irrelevant for the first item and therefore hidden by default. Since it's just invisible and still has a height it caused an unintended space above the first select when the mobile styling was applied. 

| Without `display: none` | With `display: none` |
| --- | --- |
| <img width="324" alt="Without display none" src="https://github.com/user-attachments/assets/d406f6eb-a983-4ef2-b136-d159c83b78f7"> | <img width="324" alt="With display none" src="https://github.com/user-attachments/assets/5eeb6205-1d26-47b5-87fe-b2be18c572ae"> |

Reply:
> So we should keep this?

Yes, otherwise I assume the mobile styling would be broken again

Reply:
I have kept the mobile styling just added the required Desktop styling to keep the spacing for the AND/OR field even if only a single row is used 

![image](https://github.com/user-attachments/assets/3eb0a7a8-f271-4113-a535-125c671cbcb2)